src/core/Button/index.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+
+export const Button = styled.button`
+  background-color: ${({ theme }) => theme.buttonColor};
+  border-radius: 4px;
+  border: 1px solid lightblue;
+  cursor: pointer;
+  padding: 10px;
+  & + & {
+    margin-left: 5px;
+  }
+`;
+export const DeleteButton = styled(Button)``;
```

Comment:
Are you planning to implement this? If you are going to replace the button color it's ok, remove it otherwise since it is not doing anything